substrate-node/pallets/pallet-dao/src/tests.rs

```diff
@@ -873,6 +873,37 @@ fn customize_proposal_duration_out_of_bounds_fails() {
             ),
             Error::<TestRuntime>::InvalidProposalDuration
         );
+
+        assert_noop!(
+            DaoModule::propose(
+                RuntimeOrigin::signed(1),
+                2,
```

Comment:
To facilitate flexibility I would take advantage of `MinThreshold` defined in mock file and use it in all the `propose()` calls
~~~rust
            DaoModule::propose(
                RuntimeOrigin::signed(_),
                MinThreshold::get(),
                ...
            ),
~~~

Reply:
Great suggestion.
Done